portal/src/main/webapp/js/api/cbioportal-datamanager.js

```diff
@@ -791,34 +791,28 @@ window.initDatamanager = function (genetic_profile_ids, oql_query, cancer_study_
 	    }
 	    if (attr_ids.indexOf('NO_CONTEXT_MUTATION_SIGNATURE') > -1) {
 		var mutation_signatures_promise = new $.Deferred();
+		var study_id = self.getCancerStudyIds()[0];
 		fetch_promises.push(mutation_signatures_promise.promise());
-		self.getGeneticProfiles().then(function (genetic_profiles) {
-		    var mutation_profiles = genetic_profiles.filter(function (x) {
-			return x.genetic_alteration_type === "MUTATION_EXTENDED";
-		    });
-		    $.when.apply($, mutation_profiles.map(function (mutation_profile) {
-			var _def = new $.Deferred();
-			$.ajax({
-			    type: 'POST',
-			    url: 'api-legacy/mutationsignatures',
```

Comment:
This feature has already existed, but it was temporarily disabled because it was broken, running into the out of memory error, which this pull request is intended to fix. At that time, the endpoint was `mutationsignatures`. Does that answer your question?